src/test/java/core/basesyntax/FileReaderTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.impl.CsvFileReaderImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FileReaderTest {
+    private final FileReader fileReader = new CsvFileReaderImpl();
+    private final String testFolderPath = "src/test/resources/";
+
+    @BeforeEach
+    void setUp() {
+        List<String> lines = Arrays.asList("Line 1", "Line 2", "Line 3");
+        String fileName = testFolderPath + "test.csv";
+        try {
+            Files.write(Path.of(fileName), lines);
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to set up test data: " + fileName, e);

```

Comment:
that's a very bad idea to do in the `set up` method. if you need a test file for reading - just create that file  directly